tools/rollup-es3/src/es3/FormatError.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { padEnd, isNullOrWhitespace } from "./Utils";
+import { IEs3CheckKeyword, IEs3Keyword } from "./Interfaces";
+
+export function formatError(keyword:IEs3CheckKeyword|IEs3Keyword, funcName:string, errorMsg:string, code:string, pos:number, id:string, entry:string) {
```

Comment:
Error formatter, it seems a bit over the top, but when it detects a failure this provides the context to help find the offending implementation.